src/js/twitter.js

```diff
@@ -1,3 +1,64 @@
+import { getNodesDeep } from './utils'
+
+const twitter_filter = (prev_tweet_count = 0) => {
+  const twitter_handles = []
+  const orgs = document.getElementsByClassName('org-info')
+
+  for (let i = 0; i < orgs.length; i++) {
+    const org_info = orgs[i].children
+    const children = [].slice.call(org_info)
+    const twitter_url = children.filter(
+      x => x.nodeName === 'A' && x.host === 'twitter.com'
+    )
+
+    if (twitter_url.length !== 0) {
+      const twitter_account = twitter_url[0].pathname.substr(1)
+      twitter_handles.push(twitter_account)
+    }
+  }
+
+  const iframe_id = document.getElementById('twitter-widget-0')
+
+  if (iframe_id !== null) {
+    if (typeof iframe_id.contentWindow !== 'undefined') {
+      const content_window_document = iframe_id.contentWindow
```

Comment:
Note that `The ‘content’ attribute of Window objects is deprecated. Please use ‘window.top’ instead.`.

Reply:
You've mentioned that content is deprecated and not contentWindow - are they equal to the same thing then?

Reply:
Weird, when I'm trying to reproduce it again, the warning message doesn't appear. Just let it be then.